script/build-wasm

```diff
@@ -79,7 +79,7 @@ if [ -z "$emcc" ] && [ -n "$docker" ]; then
   export PODMAN_USERNS=keep-id
   emcc="$docker run                      \
     --rm                                 \
-    -v $(pwd):/src:Z                     \
+    -v \"$(pwd)\":/src:Z                 \
```

Comment:
This gives

> docker: Error response from daemon: create ... includes invalid characters for a local volume name, only "[a-zA-Z0-9][a-zA-Z0-9_.-]" are allowed. If you intended to pass a host directory, use absolute path.

Come on, at least test your PR :)

I think you should revert this completely, quotes shouldn't be needed here